interface/billing/billing_report.php

```diff
@@ -617,9 +619,13 @@ function MarkAsCleared(Type) {
                                            title='<?php echo xla('See messages from the last set of generated claims'); ?>'><strong><?php echo xlt('View Log'); ?></strong></a>
                                     </li>
                                 <?php } ?>
+                                <li><a href="<?php echo $webroot ?>/interface/billing/customize_log.php" target="_blank"><?php  echo xlt('Tab Log') ?></a>
+                                </li>
                                 <li><a class="link_submit"
                                        href="JavaScript:void(0);" onclick="select_all(); return false;"><strong><?php echo xlt('Select All'); ?></strong></a>
                                 </li>
+                                <li><a  id="clear-log" href="#" class='btn btn-default btn-sm' title='<?php xla('Clear the log'); ?>'><?php echo xlt('Clear Log') ?></a>
+                                </li>
```

Comment:
button looks cool here but the others are button less unlike the screenshot https://community.open-emr.org/t/clearing-the-billing-log-feature/12018?u=stephenwaite

Reply:
Yes caught <li> styling change but did think to remove the button class.

Reply:
clear log just needs a strong tag
was hoping you'd add the buttons :wink:

Reply:
hi @juggernautsei, pretty sure @sjpadgett could cherry pick this once it's committed but think the clear log still needs a strong tag, thank you

Reply:
I added the stong tag now. I tried making this into buttons but it just doesn't look right. Bootstrap makes the buttons to large even if I use sm tag. And styling the ul causes even more padding to be added by the browser. The only thing that would improve the look would be to remove the bullets.